plumpy/persistence.py

```diff
@@ -581,7 +587,7 @@ def _get_value(self, saved_state, name, load_context):
         return value
 
 
-@auto_persist('_done', '_result')
+@auto_persist('_state', '_result')
```

Comment:
Just to note that this change will break what's currently in people's databases i.e. if we introduce this we should make it clear that users of AiiDA should finish all jobs in the old system and then upgrade.

One possible way to make it backwards compatible would be to, in `load_instance_state`, check for a `_done` entry and use that to set the corresponding new state.

Reply:
> One possible way to make it backwards compatible would be to, in load_instance_state, check for a _done entry and use that to set the corresponding new state.

Good idea, I'll try.

Reply:
We already merged a change that breaks existing checkpoints, but since anyway this will go in a new minor release of `plumpy` and this will require a new version of AiiDA, we don't have to guarantee that existing checkpoints continue to work after update. Before every update of AiiDA, we already tell people that they should have finished all running processes and stopped the daemon. This means that there _should_ be no active checkpoints to be restarted from. I think then it is fine to have them not backwards compatible.